.maintain/insert-session-keys.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
```

Comment:
Maybe we should add some documentation about what `$2, $3` stands for.  It would be nice to have an example here.